jvm-packages/xgboost4j/src/main/java/ml/dmlc/xgboost4j/java/Booster.java

```diff
@@ -446,14 +451,15 @@ int loadRabitCheckpoint() throws XGBoostError {
    */
   void saveRabitCheckpoint() throws XGBoostError {
     XGBoostJNI.checkCall(XGBoostJNI.XGBoosterSaveRabitCheckpoint(this.handle));
+    version += 1;
```

Comment:
we might also need to update the comments to indicate that this method update version number

Reply:
Done.